4. User Guides/README.md

```diff
@@ -15,7 +23,7 @@ There are three kinds of channels: [public channels](#public-channels), [private
 
 ### Public channels
 
-**Public channels** (also called **channels** for short) are the most important kind. Use channels for conversations that are open to your entire team. A channel can be joined by anyone on your team. 
+**Public channels** (also called **channels** for short) are the most important kind. Use channels for conversations that are open to your entire team. A channel can be joined by anyone on your team.
```

Comment:
I am too biased to be useful here.  I prefer _group_ , but indeed that will get confused with our "channel" which is also sometimes referred to as _group_    My preference for _group_ is because on Raspberry Pi or similar IoT devices deployment,  it is one-server-per-family ,  or one-server-per-special-interest-group ;  something I'm most used to in my own user of Rocket.Chat.